lib/active_merchant/billing/gateways/ipg.rb

```diff
@@ -395,6 +395,11 @@ def parse_element(reply, node)
         return reply
       end
 
+      def store_and_user_id_from(user_id)
+        split_credentials = user_id.split('._.')
```

Comment:
I didn't add any validation in here to make sure the delimiter is present. I wasn't sure if we typically validate user input to that degree, but happy to add it in if necessary.